src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -1870,12 +1873,21 @@ public void run() {
                 if (!scheduleBuilds.isEmpty()) {
                     listener.getLogger().println("Scheduling builds for branches:");
                     for (Map.Entry<P, SCMRevision> entry : scheduleBuilds.entrySet()) {
-                        listener.getLogger().println("    " + factory.getBranch(entry.getKey()).getName());
-                        if (entry.getKey().scheduleBuild(0, new SCMTrigger.SCMTriggerCause("Branch indexing"))) {
-                            try {
-                                factory.setRevisionHash(entry.getKey(), entry.getValue());
-                            } catch (IOException e) {
-                                e.printStackTrace(listener.error("Could not update last revision hash"));
+                        final P _project = entry.getKey();
+                        if (!factory.isProject(_project)) {
+                            listener.getLogger().println("Detected unsupported subitem " + _project + ", skipping");
+                            continue;
+                        }
+                        listener.getLogger().println("    " + factory.getBranch(_project).getName());
```

Comment:
Yes. It is part of a multiline message. (This code was already there, I am just reindenting basically.)